modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -175,6 +175,73 @@ definition {
 		}
 	}
 
+	@description = "LPS-165982 - Verify that the all fields from the related object are displayed when adding filters for the picklist fields to select data from the object"
```

Comment:
Use the story ticket instead of automation ticket. 
[LPS-156704](https://issues.liferay.com/browse/LPS-156704)